### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by sergiught**
### 🔧 Changes

General updates on the README. 

### 📝 Checklist

- [ ] All new/changed/fixed functionality is covered by tests (or N/A)
- [x] I have added documentation for all new/changed functionality (or N/A)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,77 +51,43 @@ provider "auth0" {}
...
 
-[[table of contents]](#table-of-contents)
 
-## Author
+---
+
+<div align="center">
 
-[Auth0](https://auth0.com/)
+<img alt="Auth0_black" src="https://user-images.githubusercontent.com/28300158/183676042-b9d92893-8fff-408f-9a36-63e77b14be30.png#gh-light-mode-only"  width="20%" height="20%">
```
Comment (willvedd) [inline README.md:82]: Alt text here should vividly describe the content of the images.

```suggestion
<img alt="Auth0 logo and wordmark in black on transparent background" src="https://user-images.githubusercontent.com/28300158/183676042-b9d92893-8fff-408f-9a36-63e77b14be30.png#gh-light-mode-only"  width="20%" height="20%">
```

File: README.md [hunk 2]
```diff
@@ -54,77 +51,43 @@ provider "auth0" {}
...
 
-[[table of contents]](#table-of-contents)
+<img alt="Auth0_white" src="https://user-images.githubusercontent.com/28300158/183676141-bea463f9-af82-40ce-b18c-3a1030183d58.png#gh-dark-mode-only"  width="20%" height="20%">
```
Comment (willvedd) [inline README.md:84]:
```suggestion
<img alt="Auth0 logo and wordmark in white on transparent background" src="https://user-images.githubusercontent.com/28300158/183676141-bea463f9-af82-40ce-b18c-3a1030183d58.png#gh-dark-mode-only"  width="20%" height="20%">
```

File: README.md [hunk 3]
```diff
@@ -54,77 +51,43 @@ provider "auth0" {}
...
+- [Auth0's code of conduct guidelines](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md)
+- [This repo's contribution guide](CONTRIBUTING.md)
 
-See [CONTRIBUTING.md](CONTRIBUTING.md).
 
-[[table of contents]](#table-of-contents)
+## 🙇 Support & Feedback
 
-## What is Auth0?
+### Raise an issue
```
Comment (willvedd) [inline README.md:66]:
```suggestion
### Raise an Issue
```

**Review by willvedd — COMMENTED** [review]
Several small suggestions, but overall I think it's a nice refresh!

**Comment by kevlil83** [comment]
README is looking good. Can we update the description to be a little more.. descriptive? For example, "The official Terraform provider for managing Auth0 management API resources." Doesn't have to be exactly that but something that expands the description a little bit. Most will prob know what a Terraform provider is but others my stumble on it and it could be a learning opportunity.

**Comment by kevlil83** [comment]
Both of the links in the documentation section link to the same page. Not sure if that is a typo.

**Comment by kevlil83** [comment]
Small thing but I would add a little bit of white space between the logo and text on the bottom.

File: README.md [hunk 4]
```diff
@@ -1,48 +1,45 @@
...
-- [Contributing](#contributing)
-- [What is Auth0?](#what-is-auth0)
-- [Create a free Auth0 Account](#create-a-free-auth0-account)
-- [Issue Reporting](#issue-reporting)
-- [Author](#author)
-- [License](#license)
+### Requirements
 
--------------------------------------
+- [Terraform](https://www.terraform.io/downloads)
```
[the remark this answers is not in the document]
Comment (sergiught) [reply README.md:28]: Not really, the provider works with any (v0 or v1).

File: README.md [hunk 5]
```diff
@@ -54,77 +51,43 @@ provider "auth0" {}
...
-If you have found a bug or if you have a feature request, please report them at this repository issues section.
 Please do not report security vulnerabilities on the public GitHub issue tracker.
 The [Responsible Disclosure Program](https://auth0.com/whitehat) details the procedure for disclosing security issues.
 
-[[table of contents]](#table-of-contents)
 
-## Author
+---
+
+<div align="center">
```
[the remark this answers is not in the document]
Comment (sergiught) [reply README.md:80]: This is following the SDK GitHub README and Docs Redesign format.

**Comment by sergiught** [comment]
Thank you both for the review @kevlil83, @willvedd. I addressed all the comments. 

@kevlil83, I changed 1 link to point now to the github repo quickstart, as the docs haven't been published to the terraform registry yet (that happens only once we make a release). Afterwards I'll update the link to point to that quickstart.

**Review by willvedd — APPROVED** [review]
👍

**Pull request merged**